Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch_description.md

```diff
@@ -0,0 +1,14 @@
+## Configure Cisco Stealthwatch on Cortex XSOAR
```

Comment:
This looks more like a readme file, you can read about the integration_description file here: https://xsoar.pan.dev/docs/documentation/integration-description
Of course @ShirleyDenkberg would know best